common/build/eslint-config-fluid/strict-ts.json

```diff
@@ -0,0 +1,1440 @@
+      "off"
+    ],
+    "@typescript-eslint/func-call-spacing": [
+      "off"
+    ],
+    "@typescript-eslint/interface-name-prefix": [
+      "off"
+    ],
+    "@typescript-eslint/keyword-spacing": [
+      "off"
+    ],
+    "@typescript-eslint/member-delimiter-style": [
+      "off"
+    ],
+    "@typescript-eslint/no-duplicate-imports": [
+      "error"
+    ],
+    "@typescript-eslint/no-dynamic-delete": [
+      "error"
+    ],
+    "@typescript-eslint/no-empty-function": [
+      "off"
+    ],
+    "@typescript-eslint/no-empty-interface": [
+      "error"
+    ],
+    "@typescript-eslint/no-extra-semi": [
+      "error"
+    ],
+    "@typescript-eslint/no-extraneous-class": [
+      "error"
+    ],
+    "@typescript-eslint/no-inferrable-types": [
+      "off"
+    ],
+    "@typescript-eslint/no-invalid-this": [
+      "off"
+    ],
+    "@typescript-eslint/no-magic-numbers": [
+      "off"
+    ],
+    "@typescript-eslint/no-require-imports": [
+      "error"
+    ],
+    "@typescript-eslint/no-shadow": [
+      "error",
+      {
+        "hoist": "all",
+        "ignoreTypeValueShadow": true
+      }
+    ],
+    "@typescript-eslint/no-this-alias": [
+      "error"
+    ],
+    "@typescript-eslint/no-throw-literal": [
+      "error"
+    ],
+    "@typescript-eslint/no-unused-expressions": [
+      "error"
+    ],
+    "@typescript-eslint/no-unnecessary-qualifier": [
+      "error"
+    ],
+    "@typescript-eslint/no-unnecessary-type-arguments": [
+      "error"
+    ],
+    "@typescript-eslint/no-use-before-declare": [
+      "off"
+    ],
+    "@typescript-eslint/no-var-requires": [
+      "error"
+    ],
+    "@typescript-eslint/object-curly-spacing": [
+      "off"
+    ],
+    "@typescript-eslint/prefer-for-of": [
+      "error"
+    ],
+    "@typescript-eslint/prefer-function-type": [
+      "error"
+    ],
+    "@typescript-eslint/prefer-readonly": [
+      "error"
+    ],
+    "@typescript-eslint/promise-function-async": [
+      "error"
+    ],
+    "@typescript-eslint/quotes": [
+      "error",
+      "double",
+      {
+        "allowTemplateLiterals": true,
+        "avoidEscape": true
+      }
+    ],
+    "@typescript-eslint/restrict-plus-operands": [
+      "error"
+    ],
+    "@typescript-eslint/restrict-template-expressions": [
+      "off"
+    ],
+    "@typescript-eslint/require-await": [
+      "off"
+    ],
+    "@typescript-eslint/semi": [
+      "error",
+      "always"
+    ],
+    "@typescript-eslint/space-infix-ops": [
+      "error"
+    ],
+    "@typescript-eslint/space-before-function-paren": [
+      "error",
+      {
+        "anonymous": "never",
+        "asyncArrow": "always",
+        "named": "never"
+      }
+    ],
+    "@typescript-eslint/strict-boolean-expressions": [
+      "error"
+    ],
+    "@typescript-eslint/triple-slash-reference": [
+      "error"
+    ],
+    "@typescript-eslint/type-annotation-spacing": [
+      "error"
+    ],
+    "@typescript-eslint/unbound-method": [
+      "off"
```

Comment:
We want this enabled.

Reply:
Please close #9082 when this gets turned on. Thanks!